Tests/ExtractFilesTests.m

```diff
@@ -220,6 +220,23 @@ - (void)testExtractFiles_NoPasswordGiven
     }
 }
 
+- (void)testExtractFiles_IncorrectPassword
+{
+    NSString *testArchiveName = @"Test Archive (Bad Password).rar";
+    NSLog(@"Testing extraction of archive %@", testArchiveName);
+    NSURL *testArchiveURL = self.testFileURLs[testArchiveName];
+    NSString *extractDirectory = [self randomDirectoryWithPrefix:
+                                  [testArchiveName stringByDeletingPathExtension]];
+    NSURL *extractURL = [self.tempDirectory URLByAppendingPathComponent:extractDirectory];
+    URKArchive *archive = [[URKArchive alloc] initWithURL:testArchiveURL password: @"password2" error: nil];
+    NSError *error = nil;
+    [archive extractFilesTo:extractURL.path
+                                 overwrite:NO
+                                     error:&error];
```

Comment:
Make sure after your code here is in a loop that the indentation for these three lines is correct (aligned on the `:`).